### PR TITLE
enabled a more agnostic way to access mapped objects in the PDO

### PR DESCRIPTION
**Pull request by af-silva**
Now the mapped objects in the PDO can be access by three ways e.g.:
node.tpdo['Statusword']
node.tpdo['0x6041']
node.tpdo[0x6041]

### CODE REVIEW

**Comment by af-silva** [comment]
@christiansandberg The travis is failing but I don't know way...

**Comment by christiansandberg** [comment]
You’re trying to add range() together but range() is an iterator in Python 3.

**Comment by af-silva** [comment]
Done! thanks! If you have a different solution please say so ;)

File: canopen/pdo/base.py [hunk 1]
```diff
@@ -30,13 +30,18 @@ def __iter__(self):
         return iter(self.map)
 
     def __getitem__(self, key):
-        if isinstance(key, int):
+        if isinstance(key, int) and key in list(set(list(range(0x1A00, 0x1A80)) + # By TPDO ID
+                                                    list(range(0x1600, 0x1680)) + # By RPDO ID
+                                                    list(range(0, 80)))):         # By PDO Index
```
Comment (christiansandberg) [inline canopen/pdo/base.py]: Hmm, maybe it's better to just check value ranges. It feels a little bit unnecessary to create a large list of valid values each time.

```python
if isinstance(key, int):
    if 0x1A00 < key < 0x1A80 or 0x1600 < key < 0x1680 or 0 < key < 80:
```

Comment (af-silva) [reply canopen/pdo/base.py]: Done.

**Comment by af-silva** [comment]
Ok everything check out now. If you can, verify the code and if you wish I think the API can be released as a version 1.0 or at least 0.9 (I don't know the working state of the local nodes implementation, so ....).

Thanks again!

**Pull request merged**